guides/common/modules/proc_host_registration_steps.adoc

```diff
@@ -0,0 +1,44 @@
+.Procedure
+
+. Generate registration command
++
+UI: In the {ProjectWebUI}, navigate to *Hosts > Register Host* and generate the registration command.
```

Comment:
This smells, @melcorr help! :-)

Reply:
Guys wants this, if you feel this needs a correction please file a PR I know you're busy.

Reply:
Filed a PR: https://github.com/theforeman/foreman-documentation/pull/828